sdk/nodejs/servicebus/zMixins.ts

```diff
@@ -323,7 +325,7 @@ export class ServiceBusFunction extends appservice.Function<ServiceBusContext, s
             connection: bindingConnectionKey,
         } as ServiceBusBindingDefinition;
 
-        const namespace = pulumi.all([namespaceName, resourceGroupName])
+        const namespace = pulumi.all([namespaceName, resourceGroupName, namespaceId, resourceGroupId])

```

Comment:
It might be worth to add a short comment of why we add IDs and then ignore them in apply, so that we don't remove this later.

Reply:
Will do.